code/modules/awaymissions/maploader/reader.dm

```diff
@@ -309,10 +321,6 @@ var/global/dmm_suite/preloader/_preloader = null
 
 	. = ..()
 
-/dmm_suite/Destroy()
-	..()
-	return QDEL_HINT_HARDDEL_NOW
-

```

Comment:
It actually only happened in the last couple of days, believe it or not. 
This was one of the last out of place Del() calls in the codebase.